src/services.js

```diff
@@ -44,6 +44,17 @@ export class Services {
         getElementServiceForDoc(nodeOrDoc, 'access', 'amp-access')));
   }
 
+  /**
+   * Returns a promise for the Embedly service.
+   * @param {!Node|!./service/ampdoc-impl.AmpDoc} nodeOrDoc
+   * @return {!Promise<!../extensions/amp-embedly/0.1/embedly-service.EmbedlyService>}
+   */
+  static embedlyServiceForDoc(nodeOrDoc) {
+    return (/** @type {!Promise<
```

Comment:
Since the service is only required at the extension level, I would suggest defining this getter in the extension. Benefit is two-fold:

- Scope is limited, so knowledge of the service doesn't spill out. This means that you can do whatever you want to the service in the future, e.g. removing it, changing API, etc.

- Knowledge of the service type remains within the extension, so you won't have to add the extension to the compile sources (which you shouldn't :))

Reply:
Updated, getter is now in the extension. 
I undid the changes on this `services.js` file as well as the one in the `compile.js` as suggested.